addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -397,4 +403,15 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
             logger.error("Error setting thing properties: {}", ex.getMessage(), ex);
         }
     }
+
+    private void validateConfiguration() {
+        boolean addressFromAutodiscovery = getThing().getProperties().containsKey(PROPERTY_ADDRESS);
+        String addressFromConfiguration = ObjectUtils.toString(getConfig().get("address"));
```

Comment:
better use a constant for "address"

Reply:
I had some quirks with the address parameter in a things file and in the discovery. In the DiscoveryResultBuilder there is only a method 'withProperty' and not 'withConfig'. So if the address has been specified in the things file, i get it with getConfig().get(...), but in the autodiscovery, i get it with getThing().getProperties()...
I will take a closer look.

Reply:
> but in the autodiscovery, i get it with getThing().getProperties()...

No - it is normal that DiscoveryResults only have properties. When creating the Thing, the framework automatically checks, which of those have to become config parameters on the Thing and the rest is added as a property.
You should thus never have an "address" property on a Thing, it will always (file + autodiscovery) be a config parameter.

Reply:
Aha, OK, thanks for explaining. Then i must have something wrong, because the address discovery property is not available as config parameter